.github/workflows/lint.yml

```diff
@@ -26,14 +26,14 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dprint/check@v2.2
         with:
-          dprint-version: '0.40.2' # selfup { "regex": "\\d[^']+", "script": "dprint --version | cut -d ' ' -f 2" }
+          dprint-version: '0.41.0' # selfup { "regex": "\\d[^']+", "script": "dprint --version | cut -d ' ' -f 2" }
```

Comment:
https://github.com/kachick/ruby-ulid/actions/runs/6527921205/job/17723418906

```
building '/nix/store/i2f0hh2px2r6cb0kcqa0sca8acd5xjaj-dprint-0.41.0.drv'...
error: builder for '/nix/store/i2f0hh2px2r6cb0kcqa0sca8acd5xjaj-dprint-0.41.0.drv' failed with exit code 101;
       last 10 log lines:
       >                 sysinfo::apple::disk::get_disks::hf0f5e9ca54811ca0 in libsysinfo-2ada6af970649dda.rlib(sysinfo-2ada6af970649dda.sysinfo.7c30ee2e47e3fa9d-cgu.12.rcgu.o)
       >             "_kCFURLVolumeAvailableCapacityKey", referenced from:
       >                 _$LT$sysinfo..apple..disk..Disk$u20$as$u20$sysinfo..traits..DiskExt$GT$::refresh::h54fb3881d5127506 in libsysinfo-2ada6af970649dda.rlib(sysinfo-2ada6af970649dda.sysinfo.7c30ee2e47e3fa9d-cgu.12.rcgu.o)
       >                 sysinfo::apple::disk::get_disks::hf0f5e9ca54811ca0 in libsysinfo-2ada6af970649dda.rlib(sysinfo-2ada6af970649dda.sysinfo.7c30ee2e47e3fa9d-cgu.12.rcgu.o)
       >           ld: symbol(s) not found for architecture x86_64
       >           clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
       >           
       >
       > warning: `dprint` (bin "dprint") generated 2 warnings
       > error: could not compile `dprint` (bin "dprint") due to previous error; 2 warnings emitted
       For full logs, run 'nix log /nix/store/i2f0hh2px2r6cb0kcqa0sca8acd5xjaj-dprint-0.41.0.drv'.
error: 1 dependencies of derivation '/nix/store/jigfmw72gszbyybcdxdygji0mblq9f9f-nix-shell-env.drv' failed to build
```

It will be fixed when https://github.com/NixOS/nixpkgs/pull/261200 will be backported into [nixos-unstable branch](https://github.com/NixOS/nixpkgs/tree/nixos-unstable)